src/components/CaculatorTable.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import './CalculatorTable.css';
+
+const CalculatorTable = () => (
+    <table>
+      <tr className='first-row'>
+        <td colSpan='4'>0</td>
+      </tr>
+      <tr className='other-rows'>
+        <td>AC</td>
+        <td>+/-</td>
+        <td>%</td>
+        <td className='last-data'>/</td>
```

Comment:
- [ ] Instead of using "/", kindly use the correct division sign as presented in the design "&div;". This would prove important for subsequent projects to be built on this one.

Reply:
Hello, I don't have '÷' on my device, my division works with '/'